tests/rules/exporter/test_rules2excel.py

```diff
@@ -0,0 +1,9 @@
+from cognite.neat.rules.exporter.rules2excel import RulesToExcel
+
+
+def test_rules2excel(simple_rules):
+    exporter = RulesToExcel(rules=simple_rules)
+    exporter.generate_workbook()
+    assert exporter.workbook.sheetnames == ["Metadata", "Classes", "Properties"]
+    assert exporter.class_counter == 4
+    assert exporter.property_counter == 9
```

Comment:
Isn't a better test to load the  excel file and check that you get the same rules out?

Reply:
it can be future improvement.